.github/workflows/c-cpp.yml

```diff
@@ -26,7 +27,8 @@ jobs:
       run: make
     - name: debuild
       run: debuild -B -us -uc
-#    - name: make check
-#      run: make check
-#    - name: make distcheck
-#      run: make distcheck
+    - name: install deb
+      run: sudo apt install ../synaptic_*.deb
```

Comment:
Probably best to add a `-y` here too so that this won't get stuck if it needs extra dependencies. Or is that not a concern (maybe I'm missing something :) ?

Reply:
Apparently it installs dependencies without the -y..  which is somewhat surprising, but it works..

Reply:
Heh, interesting. I suspect a bug in apt here but it's fine. If the behavior in apt changes we can update or workflow trivially :)